recipes/manifold/all/conanfile.py

```diff
@@ -39,20 +46,26 @@ def requirements(self):
             self.requires("onetbb/2022.0.0")
 
     def validate(self):
-        check_min_cppstd(self, 17)
+        if self.settings.compiler.get_safe("cppstd"):
+            check_min_cppstd(self, self._min_cppstd)
 
     def build_requirements(self):
         self.tool_requires("cmake/[>=3.18 <4]")
 
     def source(self):
         get(self, **self.conan_data["sources"][self.version], strip_root=True)
+        apply_conandata_patches(self)
 
     def generate(self):
         tc = CMakeToolchain(self)
         tc.cache_variables["MANIFOLD_DOWNLOADS"] = False
         tc.cache_variables["MANIFOLD_TEST"] = False
         tc.cache_variables["MANIFOLD_CBIND"] = False
         tc.cache_variables["MANIFOLD_PYBIND"] = False
+        # Introduced in 3.2.0 release but provokes error like:
+        # src/disjoint_sets.h:99:10: error: 'size_t' was not declared in this scope; did you mean 'std::size_t'
+        # Then disable it
+        tc.variables["MANIFOLD_STRICT"] = False
```

Comment:
```suggestion
        if Version(self.version) >= "3.1.0":
            tc.variables["MANIFOLD_STRICT"] = False # no -Werror
```
Introduced in 3.1.0: https://github.com/elalish/manifold/commit/b8dd573a9cd16dc043aa1295beb5741bd42eb041

Otherwise, CMake will generate warnings about non-used definition when building 3.0.1